docs/sources/tempo/setup/operator/object-storage.md

```diff
@@ -17,9 +17,7 @@ Tempo Operator supports [AWS S3](https://aws.amazon.com/), [Azure](https://azure
 
 * Create a [bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html) on AWS.
 
-### Installation
-
-* Deploy the Tempo Operator to your cluster.
+### Static Token Installation
 
 * Create an Object Storage secret with keys as follows:
```

Comment:
Since this is a procedure, the steps should be numbered instead of bullets. Ideally, this would be used throughout the document. 
```suggestion
1. Create an Object Storage secret with keys as follows:
```

Reply:
Note that line 36 below should also be changed from a bullet to a numbered step.